docs/blog/2021-12-12-log4j-zero-day-mitigation-guide.mdx

```diff
@@ -67,21 +67,29 @@ It works by scanning for hashes of [known vulnerable log4j classes](https://gith
 If you have a vulnerable version of a log4j in your built Java project, the hash will match a one
 of the hashes in the list.
 
-**[Download from GitHub](https://github.com/lunasec-io/lunasec/releases/)**
+**Download from [GitHub](https://github.com/lunasec-io/lunasec/releases/)** or from the command line:
 
-_Make sure you download the right version for your Operating System and CPU architecture._ Once downloaded, you can extract
-it and run the `log4shell` command in your terminal.  The tool can scan individual files or whole directories.
+```shell title="Download Linux CLI"
+curl -OL https://github.com/lunasec-io/lunasec/releases/download/v1.4.1-log4shell/log4shell_1.4.1-log4shell_Linux_x86_64
+```
 
+```shell title="Download OSX CLI"
+curl -OL https://github.com/lunasec-io/lunasec/releases/download/v1.4.1-log4shell/log4shell_1.4.1-log4shell_Darwin_x86_64
```

Comment:
I'd rather just link to the page.  We dont know peoples CPU arch and what about windows?